ui/tests/acceptance/task-detail-test.js

```diff
@@ -182,36 +182,6 @@ module('Acceptance | task detail', function(hooks) {
     assert.notOk(Task.hasPrestartTasks);
   });
 
-  test('the addresses table lists all reserved and dynamic ports', async function(assert) {
-    const taskResources = allocation.taskResourceIds
-      .map(id => server.db.taskResources.find(id))
-      .find(resources => resources.name === task.name);
-    const reservedPorts = taskResources.resources.Networks[0].ReservedPorts;
-    const dynamicPorts = taskResources.resources.Networks[0].DynamicPorts;
-    const addresses = reservedPorts.concat(dynamicPorts);
-
-    assert.equal(Task.addresses.length, addresses.length, 'All addresses are listed');
```

Comment:
This could be addressed (😬) separately if there’s a time crunch but I think with the removal of these tests, the `Task.addresses` and `.hasAddresses` page objects can be deleted, I couldn’t find anywhere else they were used.